packages/app-project/src/helpers/getDefaultPageProps/getDefaultPageProps.js

```diff
@@ -8,7 +8,7 @@ export default async function getDefaultPageProps({ params, query, req, res }) {
   const dismissedAnnouncementBanner = getCookie(req, 'dismissedAnnouncementBanner') || null
 
   const { props: staticProps } = await getStaticPageProps({ params, query })
-  const { project, isServer, statusCode, title, workflowID, workflows } = staticProps
+  const { project, statusCode, title, workflowID, workflows } = staticProps
```

Comment:
`isServer` wasn't being used anywhere in this file. I think it's left over from `getInitialProps`, which runs on the server and also in the browser.